test/exclude-members.test.js

```diff
@@ -413,6 +413,26 @@ test("If you're not the excluder nor the excludee then you should still be in th
     'Carol has a feed for the new key'
   )
 
+  const aliceMembers = await pull(
+    alice.tribes2.listMembers(groupId),
+    pull.collectAsPromise()
+  )
+  t.deepEquals(
+    aliceMembers.sort(),
+    [aliceRoot.id, carolRoot.id].sort(),
+    'alice gets the correct members list'
+  )
+
+  const carolMembers = await pull(
+    carol.tribes2.listMembers(groupId),
+    pull.collectAsPromise()
+  )
+  t.deepEquals(
+    carolMembers.sort(),
+    [aliceRoot.id, carolRoot.id].sort(),
+    'carol gets the correct members list'
+  )
+
```

Comment:
This might make more sense to move into list-members.test.js, we can't put absolutely everything into this file, exclusion will touch too much